exe/irb

```diff
@@ -8,4 +8,5 @@
 
 require "irb"
 
+IRB::TOPLEVEL_BINDING = binding
```

Comment:
I believe that this is a good solution to fix the problem but this assignment is so far from lib/irb/workspace.rb. Would you move the assignment to the top-level of lib/irb/workspace.rb? And, `TOPLEVEL_BINDING` is confusing with the existing constant, so I think a different name would be better.

Reply:
Agreed, a different constant name would be much less confusing.